examples/flutter_gallery/lib/demo/material/text_form_field_demo.dart

```diff
@@ -166,6 +166,26 @@ class TextFormFieldDemoState extends State<TextFormFieldDemo> {
     ) ?? false;
   }
 
+  bool _handleShortcuts(FocusNode node, RawKeyEvent event) {
+    if (event is RawKeyDownEvent) {
+      print('Scope got key event: ${event.logicalKey}, $node');
```

Comment:
We don't usually leave (console) print statements in the demos. Maybe there's something worth displaying on-screen, or maybe add a comment?